### PR TITLE
infoschema_v2: add bench test

### PR DESCRIPTION
**Pull request by ywqzzy**
### What problem does this PR solve?

Issue Number: ref #50959

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by tiancaiamao — APPROVED** [review]

**Review by zimulala — COMMENTED** [review]
Shall we record the current test results so that we can compare them in the future if there are any improvements?

**Comment by ywqzzy** [comment]
> Shall we record the current test results so that we can compare them in the future if there are any improvements?

Currently we can't get a reasonable test result since we have bottleneck when creating tables with updating bundles.
We need to optimize the update bundle part for applyDiff to get the first version of test result.

File: pkg/infoschema/bench_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+	tc := &infoschemaTestContext{
+		t:    b,
+		re:   re,
+		ctx:  kv.WithInternalSourceType(context.Background(), kv.InternalTxnDDL),
+		data: infoschema.NewData(),
+	}
+	startTime := time.Now()
+	for j := 0; j < *tableCnt; j++ {
+		logutil.BgLogger().Info("create one", zap.Int("table-idx", j))
```
Comment (zimulala) [inline pkg/infoschema/bench_test.go:72]: Do we need to remove this log？

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**